implementations/rust/ockam/ockam_command/src/identity/delete.rs

```diff
@@ -1,28 +1,34 @@
+use crate::terminal::tui::DeleteCommandTui;
 use crate::util::node_rpc;
-use crate::{docs, fmt_ok, CommandGlobalOpts};
+use crate::{docs, fmt_ok, fmt_warn, CommandGlobalOpts, Terminal, TerminalStream};
 use clap::Args;
 use colorful::Colorful;
 
+use console::Term;
 use ockam::Context;
 use ockam_api::cli_state::traits::StateDirTrait;
 
+use super::get_identity_name;
+
 const LONG_ABOUT: &str = include_str!("./static/delete/long_about.txt");
 const AFTER_LONG_HELP: &str = include_str!("./static/delete/after_long_help.txt");
 
 /// Delete an identity
 #[derive(Clone, Debug, Args)]
 #[command(
-arg_required_else_help = true,
 long_about = docs::about(LONG_ABOUT),
 after_long_help = docs::after_help(AFTER_LONG_HELP)
 )]
 pub struct DeleteCommand {
     /// Name of the identity to be deleted
-    name: String,
+    name: Option<String>,
 
     /// Confirm the deletion without prompting
     #[arg(display_order = 901, long, short)]
     yes: bool,
+
+    #[arg(long, short, group = "identities")]
```

Comment:
I would say the `group` attribute is not necessary, right? Or you added it for some specific reason?

Reply:
Following the implementation for the nodes I thought that was needed for future features.
I will remove in the new commit 🙏.